### ISSUE TITLE
tell/t/reply lookout permission [Future request]

### ISSUE THREAD

**Issue by Crypt97** [LegendOfMCPE/EssentialsPE#399]
The point is to have permission for a particular staff to see what they write in a private message.

But if a bad idea please do not close it but help me

**Comment by Sandertv** [comment]
Yep I've been thinking of this too. Will be added in the 3.0.0 rewrite.
